modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -45,22 +45,15 @@ const loadLanguage = (scriptLoader: ScriptLoader, editor: Editor) => {
   }
 };
 
-const loadTheme = (scriptLoader: ScriptLoader, editor: Editor, suffix: string): Promise<void> => {
+const loadTheme = (editor: Editor, suffix: string): void => {
   const theme = Options.getTheme(editor);
 
-  if (Type.isString(theme)) {
-    if (!hasSkipLoadPrefix(theme) && !Obj.has(ThemeManager.urls, theme)) {
-      const themeUrl = Options.getThemeUrl(editor);
-      const url = themeUrl ? editor.documentBaseURI.toAbsolute(themeUrl) : 'themes/' + theme + '/theme' + suffix + '.js';
-      ThemeManager.load(theme, url).catch(() => {
-        ErrorReporter.themeLoadError(editor, url, theme);
-      });
-    }
-
-    const waitForTheme = () => ThemeManager.waitFor(theme);
-    return scriptLoader.loadQueue().then(waitForTheme, waitForTheme);
-  } else {
-    return Promise.resolve();
+  if (Type.isString(theme) && !hasSkipLoadPrefix(theme) && !Obj.has(ThemeManager.urls, theme)) {
+    const themeUrl = Options.getThemeUrl(editor);
+    const url = themeUrl ? editor.documentBaseURI.toAbsolute(themeUrl) : 'themes/' + theme + '/theme' + suffix + '.js';
```

Comment:
While you are doing some cleanup as well would this be better as template literals? The same goes for the plugin one.

Reply:
Done and I did a couple of other minor cleanup things as well.